client-src/elements/chromedash-typeahead.ts

```diff
@@ -77,35 +75,36 @@ export class ChromedashTypeahead extends LitElement {
     this.dispatchEvent(event);
   }
 
-  reflectValue(event) {
+  reflectValue(event?) {
     if (event) {
       event.stopPropagation();
     }
-    const slInput = this.slInputRef.value;
+    const slInput = this.slInputRef.value!;
     this.value = slInput.value;
   }
 
   hide() {
-    this.slDropdownRef.value.hide();
+    this.slDropdownRef.value!.hide();
   }
 
   show() {
-    this.slDropdownRef.value.show();
+    this.slDropdownRef.value!.show();
   }
 
   findPrefix() {
-    const inputEl = this.slInputRef.value.input;
+    const inputEl = this.slInputRef.value?.input;
+    if (!inputEl) return;
     const wholeStr = inputEl.value;
     const caret = inputEl.selectionStart;
```

Comment:
Could we avoid repeatedly using `!` below by changing this to something like this?
`const caret: Number = inputEl.selectionStart || 0;`